ncm-metaconfig/src/main/metaconfig/ptpd/pan/schema.pan

```diff
@@ -0,0 +1,76 @@
+declaration template metaconfig/ptpd/schema;
+
+@documentation{
+    PTPv2 configuration based on 2.3.1 configfile
+}
+# generated based on
+# grep -B 1 'ptpengine:' /etc/ptpd2.conf | sed "s/^;\?ptpengine:\(.*\)=/    '\1' ? string # default /; s/^; *\(.*\)/    @{\1}/" | grep -v '\--'
+
+include 'pan/types';
+
+# an interface with an ip
+# TODO move to core templates
```

Comment:
tracking this in an issue https://github.com/quattor/template-library-core/issues/95 (it's a bit lame, i know, but adding more dependencies in the current pile of PRs didn't seem like a good idea, esp since i have no easy way to make new releases of template-library-core for our local usage)